Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -985,7 +985,7 @@ theorem subtypePerm_apply_zpow_of_mem {g : Perm α} {s : Finset α}
 variable [Fintype α] [DecidableEq α]
 
 /-- Restrict a permutation to its support -/
-def subtypePermOfSupport (c : Perm α) : Perm c.support :=
+def subtypePermLTSupport (c : Perm α) : Perm c.support :=
```

Comment:
This looks like a bad find and replace.

Reply:
Oh yes that's exactly what it is, annoying, sorry.

Reply:
Think I've now fixed these.